text/0000-abort-by-default.md

```diff
@@ -0,0 +1,99 @@
+- Feature Name: abort_by_default
+- Start Date: 2016-10-01
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Specify abort-by-default in `Cargo.toml` when the user does `cargo new --bin`, as well as various other refinements to the panick strategy system.
+
+# Motivation
+[motivation]: #motivation
+
+## Performance
+
+Generally, the performance of bigger programs [improves by 10%](https://www.youtube.com/watch?v=mRGb4hoGuPs), which is no small amount. When overflow checking is enabled, it is as high as 2x, making overflowing checking plausible in production.
```

Comment:
I think this might not be true. I watched the video, and I believe that Alex Crichton may have misspoke. [At 10:34](https://www.youtube.com/watch?v=mRGb4hoGuPs&feature=youtu.be&t=10m34s), he says:

> If you don't emit [landing pads] ... you can actually get 10% faster compiles as well as 10% **faster** binaries.

[emphasis added]

However, the slide behind him says:

> * 10% faster compiles
> * 10% **smaller** binaries

[emphasis added]

I'm not sure how not emiting landing pads would have that significant of an impact on performance. I think this perfomance claim may be the result of a misunderstanding.